scout/server/blueprints/institutes/templates/overview/utils.html

```diff
@@ -382,20 +385,35 @@ <h3 class="header">Institute Settings</h3>
             </td>
             <td><!-- CADD score -->
               {% if variant.cadd_score %}
-                <span class="badge bg-secondary">{{ variant.cadd_score }}</span>
+                <span class="badge bg-secondary">{{ variant.cadd_score|round(2) }}</span>
               {% else %}
                 -
               {% endif %}
             </td>
             <td><!-- Rank score -->
-              <a href="#"><span class="badge rounded-pill bg-secondary">
+              <span class="badge rounded-pill bg-secondary">
               {{ variant.rank_score }}
               {% if variant.category == 'sv' %}
                 (v. {{case.sv_rank_model_version or 'na'}})
               {% else %}
                 (v. {{case.rank_model_version or 'na'}})
               {% endif %}
-              </span></a>
+              </span>
+            </td>
+            <!-- Revel scores -->
+            {% for score in [variant.revel, variant.revel_score] %}
+            <td>
+              {% if score %}
+                <span class="badge rounded-pill bg-secondary">
+                  {{score}}
+                </span>
+              {% else %}
+                -
+              {% endif %}
+            </td>
+            {% endfor %}
+            <td><!-- SpliceAI -->
+            {{ variant.spliceai_scores|list_remove_none|join(', ') if variant.spliceai_scores else "-" }}
```

Comment:
Ah, let's cover also for variants with many genes! E.g. 
![Screenshot 2024-01-10 at 11 26 35](https://github.com/Clinical-Genomics/scout/assets/758570/367203f0-0b5f-45ac-ad41-dc916a38a958)

Reply:
Fixed:

<img width="99" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/3b8516ab-2bca-47e8-89c3-1ad89563e129">